test/EstateRegistry.js

```diff
@@ -493,13 +493,43 @@ contract('EstateRegistry', accounts => {
     })
   })
 
-  describe('land update', function() {
-    it('should not allow old owner to update land after creating an Estate', async function() {
+  describe('LAND update', function() {
+    it('should allow owner of an Estate to update LAND data', async function() {
+      await land.assignMultipleParcels([0], [1], user, sentByCreator)
+      const estateId = await createEstate([0], [1], user, sentByUser)
+      await estate.updateLandData(estateId, 0, 1, 'newValue', sentByUser)
+      const data = await land.landData(0, 1, sentByUser)
+      data.should.be.equal('newValue')
+    })
+    it('should allow operator of an Estate to update LAND data', async function() {
+      await land.assignMultipleParcels([0], [1], user, sentByCreator)
+      await createEstate([0], [1], user, sentByUser)
+      await estate.setApprovalForAll(anotherUser, true, sentByUser)
+      await estate.updateLandData(1, 0, 1, 'newValue', sentByAnotherUser)
```

Comment:
We should use `const estateId = await createEstate([0], [1], user, sentByUser)` and `estateId` here instead of using `1`